src/app.js

```diff
@@ -1,21 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
 
-module.exports = sum;
+const PORT = process.env.port || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const filePath = normalizedURL.pathname;
+  const fileName = path.basename(filePath) || 'index.html';
+  const dirName = path.dirname(filePath);
+
+  if (dirName !== '/file') {
+    res.end('The pathname has to start with "/file/"');
+
+    return;
+  }
+
+  fs.readFile(`./src/public/${fileName}`, (err, data) => {
```

Comment:
The structure wasn't defined in the task, but on all projects I've seen the `public` folder was on the same level as `src` folder. In src we usually have something that should be built or transformed somehow, while the public folder it's just a folder with files.

Reply:
Agree, know that. Unfortunately, I can't create public folder on upper level in this task, don't understand why:(